site/content/integrate/apps/authentication/oauth2/_index.md

```diff
@@ -24,9 +24,9 @@ OAuth 2.0 applications are off by default and can be enabled by the System Admin
 
 1. Log in to your Mattermost server as the System Administrator.
 2. Go to **System Console > Integrations > Integration Management**.
-3. Set [Enable OAuth 2.0 Service Provider](https://docs.mattermost.com/administration/config-settings.html#enable-oauth-2-0-service-provider) to **True**.
-4. (Optional) If you'd like to allow external applications to post with customizable usernames and profile pictures, then set [Enable integrations to override usernames](https://docs.mattermost.com/configure/configuration-settings.html#enable-integrations-to-override-usernames) and [Enable integrations to override profile picture icons](https://docs.mattermost.com/configure/configuration-settings.html#enable-integrations-to-override-profile-picture-icons) to **true**.
-5. (Optional) If you'd like to allow users on your system who are not System Admins to create OAuth 2.0 applications, then set [Restrict managing integrations to Admins](https://docs.mattermost.com/administration/config-settings.html#restrict-managing-integrations-to-admins) to **False**.
+3. Set {{< newtabref href="https://docs.mattermost.com/administration/config-settings.html#enable-oauth-2-0-service-provider" title="Enable OAuth 2.0 Service Provider" >}} to **True**.
+4. (Optional) If you'd like to allow external applications to post with customizable usernames and profile pictures, then set {{< newtabref href="https://docs.mattermost.com/configure/configuration-settings.html#enable-integrations-to-override-usernames" title="Enable integrations to override usernames" >}} and {{< newtabref href="https://docs.mattermost.com/configure/configuration-settings.html#enable-integrations-to-override-profile-picture-icons" title="Enable integrations to override profile picture icons" >}} to **true**.
+5. (Optional) If you'd like to allow users on your system who are not System Admins to create OAuth 2.0 applications, then set {{< newtabref href="https://docs.mattermost.com/administration/config-settings.html#restrict-managing-integrations-to-adminsRestrict managing integrations to Admins" title="" >}} to **False**.
```

Comment:
This link looks incorrect. I think the link title was pasted in the incorrect place.